recipe/meta.yaml

```diff
@@ -33,6 +35,9 @@ requirements:
     - mkl-service
     - cxx-compiler  # [not win]
     - m2w64-toolchain  # [win]
+    - libpython  # [win]
+    - numba  # [not linux]
```

Comment:
numba is not a pymc3 dependency.

Reply:
I had some reason, I don't remember what though.

My guess is that some test was failing without. But if so, maybe that test was optional?

Reply:
Having numba in the dependencies tricks conda into resolving a compatible of dependencies.
A while back I did really extensive testing, comparing different sets of dependencies, diffing the resulting environments and checking if PyMC3 was importing without warnings...
That's why numba is included in https://github.com/pymc-devs/pymc3/wiki/Installation-Guide-(Windows)

Reply:
I guess they just did a better job at defining the dependencies compared to the authors of Theano/Aesara conda recipes.